spec/support/shared_examples/school_form_validations.rb

```diff
@@ -18,12 +18,4 @@
       expect(subject.errors[:results_search_again_query]).to include(I18n.t("activemodel.errors.models.schools_form.attributes.query.length", count: 2))
     end
   end
-
```

Comment:
I don't think you need to delete this. Happened do this in my PR, but you might as well do it here now:

```ruby
context "with no school chosen and no query provided" do
  let(:form_name) { school_id_key.sub("id", "form") }
  let(:params) { { "results_search_again_query" => "", school_id_key => "" } }

  it "returns an error" do
    expect(subject.errors[school_id_key]).to include(
      I18n.t("activemodel.errors.models.schools/#{form_name}.attributes.#{school_id_key}.blank"),
    )
  end
end
```

Reply:
All done 👍